### PR TITLE
E2e rinkeby submit value

### PR DESCRIPTION
**Pull request by tallywiesenberg**
Working on an e2e test to write to submitValue() on rinkeby TellorX.

had to strip out `estimateGas()` for building TXs, as it was severely underestimating. This is why Tim could submit TXs from remix, but @oraclown and I couldn't from pytelliot.

### CODE REVIEW

**Comment by tallywiesenberg** [comment]
Closes #122 
Closes #124

**Comment by tallywiesenberg** [comment]
Closes #127

**Comment by tallywiesenberg** [comment]
I've replaced `write` in the tests with `write_with_retry` to prevent random tx failure issues. They arise because we submit two test TXs in the GH actions at the same time from the same address

File: src/telliot/contract/gas.py [hunk 1]
```diff
@@ -10,14 +10,16 @@ async def fetch_gas_price() -> int:
     """Estimate current ETH gas price
 
     Work In Progress - Just do something quick
+
+    ethgasstation returns gas in gwei*10
     """
     return await ethgasstation("fast")
 
 
 async def ethgasstation(style: ethgastypes = "fast") -> int:
```
Comment (oraclown) [inline src/telliot/contract/gas.py:19]: style --> speed

**Review by oraclown — APPROVED** [review]
Awesome. Looks good, Tally.

**Pull request merged**